js/index.js

```diff
@@ -285,16 +280,45 @@ $(function() {
         $('#showjs').show();
       }
     });
+    var publishNewGist = function() {
+        if (!confirm('Do you really want to publish as a Gist?')) { return; }
+
+        var data = {
+            "description": "Published with try.purescript.org",
+            "public": true,
+            "files": {
+                "Main.purs": {
+                    "content": $('#code_textarea').val()
+                }
+            }
+        };
 
+        $.ajax({
+            url: 'https://api.github.com/gists',
+            type: 'POST',
+            dataType: 'json',
+            data: JSON.stringify(data)
+        })
+            .success( function(e) {
+                console.log(e);
+                delete $.QueryString.session;
+                $.QueryString.gist = e.id;
+                $.setQueryParameters($.QueryString);
+            })
+            .error( function(e) {
+                console.warn("Gist creation failed: ", e);
```

Comment:
Maybe this should alert the user too (without the `e` part of course) and direct them to look at the console output for more info?